packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -43,13 +39,22 @@ function App(): JSX.Element {
 
   useEffect(() => {
     const initialize = async (): Promise<void> => {
-      const newAdapter = await createAzureCommunicationCallAdapter({
-        userId: fromFlatCommunicationIdentifier(userId) as CommunicationUserIdentifier,
-        displayName,
-        credential: new AzureCommunicationTokenCredential(token),
-        locator: { groupId: groupId }
-      });
-      setCallAdapter(wrapAdapterForTests(newAdapter));
+      if (params.useMockAdapter) {
+        setCallAdapter(new MockCallAdapter(state));
+      } else {
+        const displayName = verifyParamExists(params.displayName, 'displayName');
+        const token = verifyParamExists(params.token, 'token');
+        const groupId = verifyParamExists(params.groupId, 'groupId');
+        const userId = verifyParamExists(params.userId, 'userId');
+
+        const newAdapter = await createAzureCommunicationCallAdapter({
+          userId: fromFlatCommunicationIdentifier(userId) as CommunicationUserIdentifier,
+          displayName,
+          credential: new AzureCommunicationTokenCredential(token),
+          locator: { groupId: groupId }
+        });
+        setCallAdapter(wrapAdapterForTests(newAdapter));
```

Comment:
Can we always `wrapAdapterForTests`, regardless of whether it's a real or mock adapter? Branches in code where wrapping happens or not make it harder to follow.